### DISCUSSION TITLE
Remove the "Windows Powershell" title in the popup

### DISCUSSION THREAD

**Discussion by airitcs** [Windos/BurntToast#165] [General]
First off - Great work on this module - it's really useful, so thanks for publishing it.

I was wondering whether it would be possible to remove the "Windows Powershell" line from the top of the BurntToast notifications?

**Comment by Windos** [comment]
You can't remove it (at least as of Windows 10 20H2), but you can change it.

If you run `Get-StartApps` you'll see a list of applications and their "AppIDs", you can take these IDs and use them for your toast notification.

For example, let's say I wanted to attribute a toast to Teams. I look for it's AppID:

```powershell
Get-StartApp *Teams
```

```output
Name            AppID
----            -----
Microsoft Teams com.squirrel.Teams.Teams
```

I can then take the AppID of `com.squirrel.Teams.Teams` and give that to BurntToast.

```powershell
New-BurntToastNotification -AppID 'com.squirrel.Teams.Teams'
```

![Toast notification attributed to Microsoft Teams](https://user-images.githubusercontent.com/6955786/116012665-0384ed80-a680-11eb-8d8b-703d9dd37035.png)

Where this is useful is if you're using remote management software, you can attribute your toasts to that, or say you're working from the Windows Terminal app you could point your toasts at that instead of generically PowerShell.

If you decide you're always going to have your notifications come from a sepcific AppID, you can edit the config.json file in the module directory for BurntToast (just note that if the AppID has slashes in it, like the default one, you need to double them up or they won't work.)

**Comment by airitcs** [comment]
Awesome. That's perfect. Thanks.

Please excuse any spelling or grammatical errors. Sent from my phone.

________________________________


TIM WISER
CENTRALISED SERVICES LEAD
***@***.***
0115 860 2094
airit.co.uk
The Technology Hub, Unit 7,
​Interchange 25 Business Park,
​Bostocks Lane, Sandiacre,
​Nottingham, ​NG10 5QG
Air IT Ltd is a registered company in England number 5543898. Registered Office: The Technology Hub, Unit 7, Interchange 25 Business Park, ​Bostocks Lane, Sandiacre, ​Nottingham, ​NG10 5QG
​
​This message contains confidential information and is intended only for the intended recipients. If you are not an intended recipient you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. E-mail transmission cannot be guaranteed to be secure or error-free as information could be intercepted, corrupted, lost, destroyed, arrive late or incomplete, or contain viruses. The sender therefore does not accept liability for any errors or omissions in the contents of this message, which arise as a result of e-mail transmission.
​

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
